spec/decorators/confirmation_decorator_spec.rb

```diff
@@ -70,6 +70,46 @@
       ]
     end
 
+    context 'when no document code has been selected' do
+      let(:user_session) do
+        build(
+          :user_session,
+          :with_commodity_information,
+          :with_import_date,
+          :with_import_destination,
+          :with_country_of_origin,
+          :with_trader_scheme,
+          :with_certificate_of_origin,
+          :with_customs_value,
+          :with_measure_amount,
+          :with_additional_codes,
+          :with_excise_additional_codes,
+          :with_vat,
```

Comment:
We could jettison all of these other traits and just focus on the document codes?

Reply:
Are you very precious about it?